prow/plugins.yaml

```diff
@@ -2,50 +2,63 @@
 # Keys: Full repo name: "org/repo".
 # Values: List of plugins to run against the repo.
 ---
-google/cadvisor:
```

Comment:
Not sure how to get a sane deserialization with config and plugins in the same object -- no good way to do a `map[string][]string` that "soaks up" the rest of the keys that did not get used for first-class deserialization AFAICT